src/libs/ReportActionsUtils.ts

```diff
@@ -374,14 +374,21 @@ function shouldIgnoreGap(currentReportAction: ReportAction | undefined, nextRepo
  * Returns a sorted and filtered list of report actions from a report and it's associated child
  * transaction thread report in order to correctly display reportActions from both reports in the one-transaction report view.
  */
-function getCombinedReportActions(reportActions: ReportAction[], transactionThreadReportActions: ReportAction[], reportID?: string): ReportAction[] {
-    if (isEmptyObject(transactionThreadReportActions)) {
+function getCombinedReportActions(
+    reportActions: ReportAction[],
+    transactionThreadReportActions: ReportAction[],
+    reportID?: string,
+    transactionThreadReportID?: string | null,
```

Comment:
```suggestion
    reportActions: ReportAction[],
    transactionThreadReportID: string | null,
    transactionThreadReportActions: ReportAction[],
    reportID?: string,
```

I am more inclined towards making `transactionThreadReportID` mandatory like the way we have transactionThreadReportActions